website/docs/result_to_dict.md

```diff
@@ -82,8 +82,22 @@ base_object = session.query(Data).all()
 formatted_data = result_to_dict(base_object)

```

Comment:
we need to specify in the docs the callback must return the new data + if nothing is returnred then will use the original value